### ISSUE TITLE
Crash when tabbing game window in fullscreen

### ISSUE THREAD

**Issue by PavelSkellig** [truong-bui/AsyncLoadingScreen#9]
When Alt+Tab game in fullscreen your plugin crashes the game. Any ideas ho to fix this issue?

**Comment by truong-bui** [comment]
You shouldn't change the size of the windows when the loading screen is running, it's Slate's error.

**Comment by Mowinski** [comment]
But our player can ALT+Tab game when is in the loading screen - its common behaviour. Do you have any solution for it?

**Comment by Mowinski** [comment]
Yes! Unfortunnetly on production build still crash. I can send you a Steam Key to our game if you write to me on support@vulpesoft.pl :) Also I can share logs

**Comment by truong-bui** [comment]
Sent! Please share the crash logs too, I'll take a look at it
